Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
+        --build.exclude_dir "node_modules" \
+        --build.include_ext "go" \
+        --misc.clean_on_exit true
+    requires:
+      vars: [GITHUB_TOKEN, OPENAI_API_KEY]
+  test:
+    cmds:
+      - go test -v ./...
+
+  i18n-extract:
+    cmds:
```

Comment:
For the 'i18n-*' tasks, it might be beneficial to add checks before running these commands to ensure that dependencies are installed (e.g., ensuring 'goi18n' is available). This prevents the potential of running into runtime errors if the tools are not available. Consider adding a validation step or guard clause.